gateway/ln-gateway/src/lightning/ldk.rs

```diff
@@ -225,8 +225,8 @@ impl ILnRpcClient for GatewayLdkClient {
             // TODO: This is a placeholder. We need to get the actual alias from the LDK node.
             alias: format!("LDK Fedimint Gateway Node {}", self.node.node_id()),
             network: match self.node.config().network {
-                Network::Bitcoin => "main",
-                Network::Testnet => "test",
+                Network::Bitcoin => "bitcoin",
```

Comment:
You could just use the `Display` impl, i.e. `.to_string()`

Reply:
Done.